src/components/Label/Label.scss

```diff
@@ -25,12 +26,14 @@ $transition-timing-function: ease-in-out;
     isolation: isolate;
 
     &__text {
-        @include mixins.text-body-1();
         display: flex;
         align-items: baseline;
-        margin: 0 var(--_--margin-inline);
+        margin: 0 var(--g-label-margin-inline, var(--_--margin-inline));
```

Comment:
It can me named `--g-label-padding` like in Button's CSS API to be consistent, maybe also change prop from margin to padding

Reply:
> It can me named `--g-label-padding` like in Button's CSS API to be consistent, maybe also change prop from margin to padding

should I definitely change margin to padding? with padding, margin will be larger due to collapse margins

Reply:
No sure now, I see it conflicts with the addons space